.github/workflows/ci.yml

```diff
@@ -0,0 +1,23 @@
+name: CI
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+    branches:
+      - master
+jobs:
+  ci:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: Setup Node.js
+        uses: actions/setup-node@v1
+        with:
+          node-version: '12.x'
+      - name: Install dependencies
+        run: npm install
```

Comment:
Travis was doing `npm ci` (though, this should probably mirror how Netlify works?)

Reply:
Ah, good catch! Since we have a lock file, this will speed up CI. Will update.